### ISSUE TITLE
Custom element doesn't exist: ha-card-weather-conditions.

### ISSUE THREAD

**Issue by mfabiani53** [r-renato/ha-card-weather-conditions#65]
Just after updating to the latest release i get that error also after clearing caches many times...
How to downgrade to the previous release?
EDIT: i am reading also other users have the same issue.....

**Comment by InToSSH** [comment]
Yes I can also confirm having the same issue after the update, tried reinstall, delete/add resources, HA restart, clear browser cache, nothing helps.

**Comment by Hilo4321** [comment]
Same issue here. Same troubleshooting steps as InToSSH.

**Comment by r-renato** [comment]
Hi, can you post here the browser console logs?

**Comment by mfabiani53** [comment]
You mean this:

![Schermata 2021-01-11 alle 19 36 56](https://user-images.githubusercontent.com/37701688/104223833-73aa9300-5444-11eb-9405-47c2f1f25102.png)

**Comment by mfabiani53** [comment]
I renamed inside transl folder **no-no.json** to **no-NO.json** file and now all is working again...

**Comment by Hilo4321** [comment]
> I renamed inside transl folder **no-no.json** to **no-NO.json** file and now all is working again...

Thank you! Worked for me as well.

**Comment by r-renato** [comment]
Sorry, bug fixed.

**Comment by mfabiani53** [comment]
Ok, now we need to rename again the file or not?

**Comment by r-renato** [comment]
no

**Comment by testorp** [comment]
I'm new to HA have installed the latest version of the add on but still get this message
how to access log_files to see what is going on?
![image](https://user-images.githubusercontent.com/44940599/106777610-6e0c2b80-6645-11eb-9a50-6fd7444b6bd1.png)
